### ISSUE TITLE
Expected Test Failure in Notebook Unit Tests

### ISSUE THREAD

**Issue by rsudarson** [zonca/pytest-ipynb#24]
I have some basic tests where each cell calls a function with some arguments. I need to write some negative test-cases where  the test  is expected to fail.

**Sample Test:**
options={}
data = getData(options)

When I execute the notebook tests how should I  expect failure for a negative test-case  and treat the cell execution as pass. Do we have any annotations for handling this at cell level.

**Comment by zonca** [comment]
no, that is not implemented, if you are interested in implementing it I
would be happy to review your pull request.
thanks
